blead/patches/001.patch

```diff
@@ -173,8 +160,36 @@ index 552a106..a72a7c8 100644
  #ifdef PERL_MICRO
  #   include "uconfig.h"
  #else
+diff --git a/iperlsys.h b/iperlsys.h
+index b922af0..daed107 100644
+--- a/iperlsys.h
++++ b/iperlsys.h
+@@ -777,21 +777,8 @@ struct IPerlLIOInfo
+ #    define PerlLIO_lstat(name, buf)    PerlLIO_stat((name), (buf))
+ #  endif
+ #  define PerlLIO_mktemp(file)          mktemp((file))
+-#  if defined(OEMVS)
```

Comment:
do you pull in the additional library now? If so, does that code need to show up in the patch?

Reply:
Yeah, this is just adjusting an old patch so that it applies properly.  The library is in 006.patch.  I'll rework the patches when I migrate to the new framework